popup.js

```diff
@@ -0,0 +1,53 @@
+function msToTime(s) {
```

Comment:
Use http://www.w3schools.com/jsref/jsref_obj_date.asp instead of manual math.
Something like:
    var date = new Date();
    date.setMilliseconds(ms);
    date.toISOString().substr(11, 8);